Clients/Xamarin.Interactive.Client.Web/Hubs/InteractiveSessionHub.cs

```diff
@@ -80,22 +84,31 @@ public IObservable<InteractiveSessionEvent> ObserveSessionEvents ()
         {
             var events = GetSession ().Events;
 
-            if (hostingEnvironment.IsDevelopment ()) {
-                var settings = new Serialization.ExternalInteractiveJsonSerializerSettings ();
-                events.Subscribe (new Observer<InteractiveSessionEvent> (evnt => {
+            events.Subscribe (new Observer<InteractiveSessionEvent> (evnt => {
```

Comment:
It seemed silly to have two different subscriptions, so I moved the `if` check inside this one.